recipes/automake/all/conanfile.py

```diff
@@ -7,7 +7,7 @@
 from conan.tools.layout import basic_layout
 from conan.tools.scm import Version
 
-required_conan_version = ">=1.53.0"
+required_conan_version = ">=1.55.0"
```

Comment:
why?

Reply:
Hi @Spacelm,
No, it's not strictly necessary, but it is a convenient field to touch to get the CI to re-run when it gets stuck. Also, specifying an older version than what the CI is running with means that there is no guarantee that the change will work with the specified version.
And yes, the other changes were subsumed/augmented in an overlapping change. I've updated the description to the current status. Thanks

Reply:
min version is a nice to have, dropping older version is allowed, and optional patches should not have been there... all of that sounds totally acceptable :)

Reply:
I just say this PR is not anymore about conan 2.0 compatibility. And I disagree with changing min conan version, specially when we know it was totally fine.